buildSrc/src/main/java/Dependencies.kt

```diff
@@ -54,7 +54,7 @@ object Versions {
         const val test = "1.3.0"
         const val test_ext = "1.1.2"
         const val espresso = "3.3.0"
-        const val room = "2.3.0"
+        const val room = "2.4.0-beta02"
```

Comment:
Just to confirm that this changes was intentional ? :)

Reply:
Ah, my bad. This is what I get for submitted a PR right at the end of my day.

This change doesn't need to be accepted, but is required for A-C to build correctly on M1 macs. What level of care is generally taken for updating dependencies in this project? Are we more careful about alpha and beta artifacts?

Reply:
Yeah, we try to only use stable dependencies. Other than that, updating to the latest stable version is great - but needs to be thoroughly tested since it will update it for downstream projects too.

Reply:
Cool, that makes sense. I'll revert this change